Box.V2/Models/BoxSignTemplate.cs

```diff
@@ -0,0 +1,128 @@
+        public const string FieldAreRecipientsLocked = "are_recipients_locked";
+        public const string FieldCustomBranding = "custom_branding";
+        public const string FieldDaysValid = "days_valid";
+        public const string FieldEmailMessage = "email_message";
+        public const string FieldEmailSubject = "email_subject";
+        public const string FieldName = "name";
+        public const string FieldParentFolder = "parent_folder";
+        public const string FieldReadySignLink = "ready_sign_link";
+        public const string FieldSigners = "signers";
+        public const string FieldSourceFiles = "source_files";
+
+        /// <summary>
+        /// Additional information on which fields are required and which fields are not editable.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldAdditionalInfo)]
+        public virtual BoxSignTemplateAdditionalInfo AdditionalInfo { get; private set; }
+
+        /// <summary>
+        /// Indicates if the template email settings are editable or not.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldAreEmailSettingsLocked)]
+        public virtual bool AreEmailSettingsLocked { get; private set; }
+
+        /// <summary>
+        /// Indicates if the template input fields are editable or not.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldAreFieldsLocked)]
+        public virtual bool AreFieldsLocked { get; private set; }
+
+        /// <summary>
+        /// Indicates if the template files are editable or not. This includes deleting or renaming template files.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldAreFilesLocked)]
+        public virtual bool AreFilesLocked { get; private set; }
+
+        /// <summary>
+        /// Indicates if the template document options are editable or not, for example renaming the document.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldAreOptionsLocked)]
+        public virtual bool AreOptionsLocked { get; private set; }
+
+        /// <summary>
+        /// Indicates if the template signers are editable or not.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldAreRecipientsLocked)]
+        public virtual bool AreRecipientsLocked { get; private set; }
+
+        /// <summary>
+        /// Custom branding applied to notifications and signature requests.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldCustomBranding)]
+        public virtual BoxSignTemplateCustomBranding CustomBranding { get; private set; }
+
+        /// <summary>
+        /// Set the number of days after which the created signature request will automatically expire if not completed.
+        /// By default, we do not apply any expiration date on signature requests, and the signature request does not expire.
+        /// </summary>
+        [JsonProperty(PropertyName = FieldDaysValid)]
+        public virtual int? DaysValid { get; private set; }
```

Comment:
Is there a reason this property is set as nullable and all the bools are non-nullable?